### PR TITLE
Createprofile, Revamp settings, NEW User dropdown component 

### PR DESCRIPTION
**Pull request by thomasyeung687**

### CODE REVIEW

File: components/utility/nostr-helper-functions.ts [hunk 1]
```diff
@@ -33,7 +39,7 @@ export async function PostListing(
         ["d", "30402"],
         [
           "a",
-          "31990:" + decryptedNpub + ":" + dValue,
+          "31990:" + userNPub + ":" + dValue,
```
Comment (calvadev) [inline components/utility/nostr-helper-functions.ts]: should be userPubkey

File: components/utility/nostr-helper-functions.ts [hunk 2]
```diff
@@ -71,7 +77,7 @@ export async function PostListing(
         "Content-Type": "application/json",
       },
       data: {
-        pubkey: decryptedNpub,
+        pubkey: userNPub,
```
Comment (calvadev) [inline components/utility/nostr-helper-functions.ts]: should be userPubkey

File: components/utility/nostr-helper-functions.ts [hunk 3]
```diff
@@ -83,7 +89,7 @@ export async function PostListing(
     });
     return {
       id: res.data.id,
-      pubkey: decryptedNpub,
+      pubkey: userNPub,
```
Comment (calvadev) [inline components/utility/nostr-helper-functions.ts]: should be userPubkey

**Review by ericspaghetti — APPROVED** [review]
🚢

File: components/home/my-listings.tsx [hunk 4]
```diff
@@ -2,23 +2,21 @@ import router from "next/router";
...
-import { Button } from "@nextui-org/react";
+import { Button, useDisclosure } from "@nextui-org/react";
 import { SHOPSTRBUTTONCLASSNAMES } from "../utility/STATIC-VARIABLES";
 import SignInModal from "../sign-in/SignInModal";
 
 export const MyListingsPage = () => {
-  let usersNPub = getLocalStorageData().decryptedNpub;
-  const [openSignInModal, setOpenSignInModal] = useState(false);
-  let [count, setcount] = useState(0);
+  let usersNPub = getLocalStorageData().userNPub;
```
Comment (calvadev) [inline components/home/my-listings.tsx]: this should be changed to const usersPubkey = getLocalStorageData().userPubkey;, and usages of usersNpub should be changed as  to usersPubkey

**Review by calvadev — APPROVED** [review]

**Pull request merged**